test/io-tests/configs/app-settings.config

```diff
@@ -6,3 +6,4 @@ server-host = "127.0.0.1"
 server-port = 49421
 
 app.settings.external_api_secret = "0123456789abcdef"
+raw-output-media-types=["image/jpeg","image/png"]
```

Comment:
New feature should not affect other tests configs. One dedicated config should be added.

Reply:
You should probably have one new config with `text/html` added to the list. We already have a test for that type somewhere, and that one should pass.